pkg/rest/core/fake/rest_client_test.go

```diff
@@ -214,3 +207,45 @@ func TestGetItems(t *testing.T) {
 	}
 
 }
+
+func TestCreateItem(t *testing.T) {
+	testCases := []struct {
+		name           string
+		storage        NamespacedStorage
+		watcher        *Watcher
+		rw             *responseWriter
+		url            string
+		expectedStatus int
+	}{
+		{"Create Item (empty storage)", make(NamespacedStorage), NewWatcher(), newResponseWriter(), fmt.Sprintf("/apis/servicecatalog.k8s.io/v1alpha1/namespaces/%s/%s", ns1, tipe1), http.StatusCreated},
+		// {"Create Item(non-empty storage)", createMultipleItemStorage(), NewWatcher(), newResponseWriter(), fmt.Sprintf("/apis/servicecatalog.k8s.io/v1alpha1/namespaces/%s/%s", ns1, tipe1), http.StatusOK},
+	}
+
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			request, err := http.NewRequest("POST", tc.url, nil)
```

Comment:
I still need to put the object in the body. Not sure how yet. Would something like this work? 
` /Users/crismarmejia/go/src/github.com/kubernetes-incubator/service-catalog/pkg/rest/core/fake/rest_client.go:321 +0x2b8`

ps. I've noticed that users of the rest_client.go are not using the router but the methods on storage directly. Should I stop testing these functions and mark as the issue as completed(and move to other things)?